test/integration/quota/quota_test.go

```diff
@@ -477,56 +473,65 @@ func TestQuotaLimitService(t *testing.T) {
 			},
 		},
 	}
+
 	waitForQuota(t, quota, clientset)
 
-	tests := []testCase{
```

Comment:
I'm generally in favor of using test tables, but here I removed it because each test case is dependant on the state created from previous test cases.